network/network_test.go

```diff
@@ -334,3 +328,77 @@ func TestInvalidRelayAddress(t *testing.T) {
 	_, err = NewNetwork(conf)
 	assert.Error(t, err)
 }
+
+func TestConnections(t *testing.T) {
+	t.Parallel() // run the tests in parallel
+
+	ts := testsuite.NewTestSuite(t)
+
+	tests := []struct {
+		bootstrapAddr string
+		peerAddr      string
+	}{
+		{"/ip4/127.0.0.1/tcp/%d", "/ip4/127.0.0.1/tcp/0"},
```

Comment:
I think if we use 127.0.0.1 for tests it perfectly works on shell pipeline runners but in docker pipeline runners it would be better to use 0.0.0.0 which work's like 127.0.0.1 in shell runners.

Reply:
Same as above ☝️